TestModels/SimpleTypes/SimpleInteger/runtimes/rust/src/types.rs

```diff
@@ -0,0 +1,58 @@
+// Code generated by software.amazon.smithy.rust.codegen.smithy-rs. DO NOT EDIT.
+
+#[derive(::std::clone::Clone, ::std::fmt::Debug)]
+pub struct SimpleIntegerConfig {
+    // XXX : ::std::option::Option<XXXtype>
+}
+
+impl SimpleIntegerConfig {
+    pub fn builder() -> SimpleIntegerConfigBuilder {
+	SimpleIntegerConfigBuilder::new()
+    }
+
+    pub fn to_internal(&self) -> simple_integer_dafny::_simple_dtypes_dinteger_dinternaldafny_dtypes::SimpleIntegerConfig {
```

Comment:
I think it would be better to define these inside the `conversions` module instead, just as we do for operation inputs, outputs, and error shapes. That will keep `SimpleIntegerConfig` as a simple datatype that doesn't depend on implementation details (in this case the Dafny implementation, but even the other smithy code generators try hard to keep protocol details decoupled from shapes).